react-native-libraries.json

```
@@ -12143,5 +12143,16 @@
     "ios": true,
     "android": true,
     "expoGo": true
+  },
+  {
+    "githubUrl": "https://github.com/Netizen-Teknologi/react-native-maps-leaflet",
+    "npmPkg": "https://www.npmjs.com/package/@netizen-teknologi/react-native-maps-leaflet",
+    "examples": [
+      "https://github.com/Netizen-Teknologi/react-native-maps-leaflet",
+    ],
```

Hello @rinaldycodes, thanks for adding new entry to the directory! 👍 

Raw JSON cannot have tailing coms, thus why CI has failed, also the `npmPkg` need to be set to npm package name, not registry the URL:

```suggestion
    "npmPkg": "@netizen-teknologi/react-native-maps-leaflet",
    "examples": [
      "https://github.com/Netizen-Teknologi/react-native-maps-leaflet"
    ],
```

Besides that the entry looks good to me!

---

thank you @Simek for your review